include/aruku/config/node/config_node.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#ifndef ARUKU__CONFIG__NODE__CONFIG_NODE_HPP_
+#define ARUKU__CONFIG__NODE__CONFIG_NODE_HPP_
+
+#include <memory>
+#include <string>
+
+#include "aruku/config/utils/config.hpp"
+#include "aruku_interfaces/msg/set_config.hpp"
+#include "aruku_interfaces/srv/get_config.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+namespace aruku
+{
+
+class ConfigNode
+{
+public:
+  explicit ConfigNode(rclcpp::Node::SharedPtr node, const std::string & path);
+
+private:
+  std::string get_node_prefix() const;
```

Comment:
Let me know why did you suggest that?

Reply:
because it return a constant value, and won't change, every instance of ConfigNode will return the same value. Or you could just make that function as a static variable, or if it's rarely used, just remove this function.

<img width="361" alt="image" src="https://user-images.githubusercontent.com/10202888/162146964-78157318-f82b-4b93-a91a-f38b3546e2ee.png">